templates/zerver/about.html

```diff
@@ -0,0 +1,124 @@
+  documentation</a>.
+</p>
+
+<h2>Zulip Community</h2>
+
+<p>
+  Zulip is developed by a vibrant community of developers from around
+  the world.  Every month, dozens of people contribute code to Zulip,
+  and dozens more contribute bug reports, feedback, and translations.
+  The Zulip community welcomes new contributors from any background.
+  The Zulip project has an easy to install development environment, an
+  extensive test suite,
+  and <a href="http://zulip.readthedocs.io/en/latest/">over 50,000
+  words of developer documentation</a> to make it easy for new
+  contributors to contribute effectively to the project.
+</p>
+
+<h3>Contributing to Zulip</h3>
+
+<p>
+  If you'd like to join the Zulip community and contribute to Zulip,
+  we'd love to have you!  Please
+  visit <a href="https://github.com/zulip/zulip/#zulip-overview">the
+  main Zulip project on GitHub</a> for details on how to get involved!
+</p>
+
+<h2>Zulip history</h2>
+
+<p>
+  Zulip was originally developed by Zulip, Inc., a small startup in
+  Cambridge, Massachusetts.  Zulip, Inc. was founded by the MIT team
+  that previously created
+  <a href="https://www.ksplice.com">Ksplice</a>, software for
+  live-patching a running Linux kernel.  Zulip was inspired by
+  the <a href="https://barnowl.mit.edu/">Barnowl</a> client for
+  the <a href="https://en.wikipedia.org/wiki/Zephyr_(protocol)">Zephyr</a>
+  protocol, and the incredible community that Zephyr supported at MIT.
+</p>
+
+<p>
+  Zulip, Inc. was acquired by Dropbox in early 2014, while the product
+  was still in private beta. Zulip's beta
+  users <a href="https://www.recurse.com/blog/90-zulip-supporting-oss-at-the-recurse-center">loved
+  Zulip's unique user experience</a> and continued
+  using Zulip despite the fact that Zulip was not being actively
+  developed.  After a year and a half, Dropbox generously decided to release Zulip as open
+  source software so that Zulip's users could continue
+  enjoying the software.
+</p>
+
+<p>
+  As a result, the first time the public had the opportunity to use
+  Zulip was when Dropbox
+  <a href="https://blogs.dropbox.com/tech/2015/09/open-sourcing-zulip-a-dropbox-hack-week-project/">released
+  Zulip as open source software</a> in late 2015.  The Zulip open
+  sourcing announcement was very popular, staying at the top of
+  both <a href="https://news.ycombinator.com/item?id=10279961">Hacker
+  News</a>
+  and <a href="https://www.reddit.com/r/programming/comments/3me9qp/dropbox_has_open_sourced_zulip_group_chat_software/">the
+    programming subreddit</a> for an entire day.
+</p>
+
+<p>
+  Zulip was open sourced with the complete version control history
+  intact because 10 Zulip users visited Dropbox for a full week to
+  help with the technical work required to open source Zulip well.
+  The Zulip community is incredibly grateful to both Dropbox and those
+  enthusiastic early users of Zulip for making the Zulip open source
+  project possible.
```

Comment:
Suggest:

> Zulip was open-sourced with its version control history intact because 10 users visited Dropbox for a full week to help with the technical work required. The community is incredibly grateful to both Dropbox and those enthusiastic early users for making the open source project possible.